File: orc8r/cloud/go/services/materializer/gateways/metrics/metrics.go
/*
Copyright (c) Facebook, Inc. and its affiliates.
All rights reserved.

This source code is licensed under the BSD-style license found in the
LICENSE file in the root directory of this source tree.
*/

package metrics

import "github.com/prometheus/client_golang/prometheus"

var (
	UpdatesProcessed = prometheus.NewCounter(prometheus.CounterOpts{
		Name: "gateway_state_updates_processed",
		Help: "Number of gateway state updates processed by the gateways materializer state recorder",
	})
)

func init() {
	prometheus.MustRegister(UpdatesProcessed)
}

File: example/gateway/python/magmaexample/hello/metrics.py
"""
Copyright (c) 2016-present, Facebook, Inc.
All rights reserved.

This source code is licensed under the BSD-style license found in the
LICENSE file in the root directory of this source tree. An additional grant
of patent rights can be found in the PATENTS file in the same directory.
"""

from prometheus_client import Counter

# Define a new counter, which can be incremented in the rpc servicer
NUM_REQUESTS = Counter('num_requests', 'Total requests')
